tests/e2e/performance/test_vdbench_workload.py

```diff
@@ -198,6 +190,44 @@ def test_vdbench_workload(
         sf_data = templating.load_yaml(template)
 
         target_results = template + 'Results'
+
+        log.info('Calculating Storage size....')
+        ceph_cluster = CephCluster()
+        total_capacity = ceph_cluster.get_ceph_capacity()
+        assert (total_capacity > constants.VDBENCH_MIN_CAPACITY), \
+            "Storage capacity is too low for performance testing"
+        log.info(f'The Total usable capacity is {total_capacity}')
+
+        if load:
+            width = constants.VDBENCH_WIDTH
+            depth = constants.VDBENCH_DEPTH
+            file_size = constants.VDBENCH_FILE_SIZE
+            capacity_per_pod = constants.VDBENCH_CAP_PER_POD
+            total_dirs = width ** depth
+            log.info(f'The total dirs in the tree {total_dirs}')
+            log.info(f'Going to run with {load} % of the capacity load.')
+            tested_capacity = round(total_capacity * 1024 * load / 100)
+            log.info(f'Tested capacity is {tested_capacity} MB')
+            servers = round(tested_capacity / capacity_per_pod)
+
+            """
+                To spread the application pods evenly on all workers or application nodes and at least 2 app pods
```

Comment:
Text and quotation marks are not aligned

Reply:
fixed.